tests/test_resolve.py

```diff
@@ -18,10 +18,6 @@
 from conda.testing.helpers import TEST_DATA_DIR, add_subdir, add_subdir_to_iter, \
     get_index_r_1, get_index_r_4, raises
 
```

Comment:
It turns out this test was duplicated in the pluggable solvers refactor, but this original was not removed. Another branch tests removing this file, which would get rid of a lot of seemingly spontaneous test failures.